model/task/task.go

```diff
@@ -661,7 +661,7 @@ func (t *Task) IsPatchRequest() bool {
 
 // IsUnfinishedSystemUnresponsive returns true only if this is an unfinished system unresponsive task (i.e. not on max execution)
 func (t *Task) IsUnfinishedSystemUnresponsive() bool {
-	return t.isSystemUnresponsive() && t.Execution < evergreen.MaxTaskExecution
+	return t.isSystemUnresponsive() && t.Execution < evergreen.GetEnvironment().Settings().TaskLimits.MaxTaskExecution
```

Comment:
It could be worth keeping MaxTaskExecution as 9 in the codebase as an absolute failsafe in case this value ever gets removed from admin settings (and to maintain consistency until you're able to set the value)